sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -633,4 +634,15 @@ class AnalysisSuite extends AnalysisTest with Matchers {
     val res = ViewAnalyzer.execute(view)
     comparePlans(res, expected)
   }
+
+  test("SPARK-28002: CTE with non-existing column alias") {
```

Comment:
For new feature test case, we don't use SPARK issue id. Let's remove this `SPARK-28002: `.

Reply:
thanks, fixed